packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -2324,26 +2324,30 @@ class WebpackCLI {
       return;
     }
 
-    const signals = ["SIGINT", "SIGTERM"];
-    let needForceShutdown = false;
-
-    signals.forEach((signal) => {
-      const listener = () => {
-        if (needForceShutdown) {
-          process.exit();
-        }
+    if (compiler.cache) {
```

Comment:
It can be multi compiler

Reply:
Please check in latest commit, I updated it to use options.